src/SFML/Window/Context.cpp

```diff
@@ -48,7 +48,7 @@ namespace sf
 ////////////////////////////////////////////////////////////
 Context::Context() : m_context(priv::GlContext::create())
 {
-    if (!setActive(true))
```

Comment:
Based on [your forum post](https://en.sfml-dev.org/forums/index.php?topic=29252.0) did you mean to put this check in the destructor?